src/components/Banners/Implementations/WritersCohortBanner.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react"
+import { Text } from "@chakra-ui/react"
+
+import DismissableBanner from "../DismissableBanner"
+import Link from "../../Link"
+
+const WritersCohortBanner = () => {
+  if (
+    typeof window !== `undefined` &&
+    window.location.pathname.includes("contributing")
```

Comment:
For SSR purposes, a better way of getting the `pathname` is from Gatsby itself (or specifically from reach router which is what they use underneath).

```tsx
// every page or template receives a `location` prop
const StaticPage = ({ data, location }) => {
  ...
}

// then you can pass that prop down to the components
<WritersCohortBanner pathname={location.pathname} />
```

This way you abstract yourself from the `window` object and you can render the same thing server and client side.

Reply:
Makes sense, changed this PR over to this just now.